src/storage/src/source/reclock.rs

```diff
@@ -295,7 +461,13 @@ impl ReclockOperator {
     /// When this function returns the local dTVC view of the remap collection will contain
     /// definite timestamp bindings that can be used to reclock messages at offsets that are not
     /// beyond the provided frontier.
-    async fn mint<P: Borrow<PartitionId>>(&mut self, source_frontier: &HashMap<P, MzOffset>) {
+    pub async fn mint<P: Borrow<PartitionId>>(
+        &mut self,
+        source_frontier: &HashMap<P, MzOffset>,
+    ) -> Vec<(PartitionId, Vec<(u64, MzOffset)>)> {
```

Comment:
Can we use the type alias `Timestamp` in all places where a `u64` is used as a timestamp?

Reply:
This has been there since the big bang, but will change. :sweat_smile: